src/App.js

```diff
@@ -9,7 +11,12 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <h1>React Article</h1>
```

Comment:
Page should usually have only one `h1` element. So, this header can be safely deleted.